tests/Elastic.CommonSchema.Tests/Specs/Spec.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using FluentAssertions;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Elastic.CommonSchema.Tests.Specs
+{
+	public static class Spec
+	{
+		private const string CommitHash = "868cdfb178bc1959003c62aefed4d5b0c10af4c0";
```

Comment:
If this could tie in with:  `./build.sh update-logging-spec`  updating the logging spec becomes easy as pie.

Reply:
done!